src/website/views/components/toolbar/_examples.pug

```diff
@@ -1187,9 +1463,47 @@ h2 Examples
             .chi-divider.-vertical
             chi-link(href='#', size='md') Save View
           .chi-toolbar__end
-            button.chi-button.-icon.-primary.-flat.-no-hover(aria-label="Button action")
+            button.chi-button.-icon.-flat(aria-label="Button action")
               .chi-button__content
                 i.chi-icon.icon-x(aria-hidden="true")
+      .chi-bulk-actions
+        .chi-bulk-actions__top
+          chi-link(href='#') Select all
+          chi-link(href='#') Clear
+        .chi-bulk-actions__start
+          .chi-bulk-actions__results
+            .chi-bulk-actions__label Actions (10 Selected)
+            .chi-form__item
+              .chi-checkbox
+                input#checkbox-ba1.chi-checkbox__input(type='checkbox')
+                label.chi-checkbox__label(for='checkbox-ba1') Show Selected Only
+          .chi-bulk-actions__buttons
+            .chi-bulk-actions__buttons-mobile
+              chi-button(variant='flat' type='icon' alternative-text='Button action' data-tooltip='Edit')
+                chi-icon(icon='edit')
+              chi-button(variant='flat' type='icon' alternative-text='Button action' data-tooltip='Compose')
+                chi-icon(icon='compose')
+              chi-button(variant='flat' type='icon' alternative-text='Button action' data-tooltip='Delete')
+                chi-icon(icon='delete')
+              chi-button(variant='flat' type='icon' alternative-text='Button action' data-tooltip='Print')
+                chi-icon(icon='print')
+            .chi-bulk-actions__buttons-desktop
+              .chi-divider.-vertical
```

Comment:
@jllr Thoughts on placing the divider between `.chi-bulk-actions__results` and `.chi-bulk-actions__buttons` ?
It would be still hidden on mobile considering CSS rules you applied to `.chi-divider`